packages/pymotif/pymotif/tests/test_motif.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Cylynx.
+# Distributed under the terms of the Modified BSD License.
+
+import json
+
+from ..motif import Motif
+from .. import constants as C
+from . import constants as TC
```

Comment:
what does `TC` stand for? test constants?

Reply:
Yep, shall add a comment there the next time I commit